src/qt/sendcoinsdialog.cpp

```diff
@@ -112,6 +112,7 @@ SendCoinsDialog::SendCoinsDialog(const PlatformStyle *_platformStyle, QWidget *p
     ui->groupCustomFee->button((int)std::max(0, std::min(1, settings.value("nCustomFeeRadio").toInt())))->setChecked(true);
     ui->customFee->setValue(settings.value("nTransactionFee").toLongLong());
     ui->checkBoxMinimumFee->setChecked(settings.value("fPayOnlyMinFee").toBool());
+    ui->optInRBF->setCheckState(fWalletRbf ? Qt::Checked : Qt::Unchecked);
```

Comment:
nit: we should probably access `fWalletRbf` over `WalletModel` (not directly over the global space).